utils/compare_minions.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+
+import salt.config
+import salt.loader
+
+import json
+import json_delta
+
+try:
+    from colorama import Fore, Back, Style, init
+    init()
+except ImportError:  # fallback so that the imported classes always exist
+    class ColorFallback():
+        __getattr__ = lambda self, name: ''
+    Fore = Back = Style = ColorFallback()
```

Comment:
Fore is Back and is Style and this is the same object for all 3 variables. This is ok for this case. But such approach is error prone in general.